app/charts/index.ts

```diff
@@ -480,7 +480,14 @@ const CHART_ADJUST_CONFIG: {
         oldValue,
         newChartConfig,
       }: FieldAdjustParams<ColumnConfig>) => {
-        newChartConfig.fields.segment = oldValue;
+        newChartConfig.fields.segment = {
+          ...oldValue,
+          type: "stacked",
+          sorting: oldValue.sorting || {
+            sortingOrder: "asc",
+            sortingType: "byDimensionLabel",
+          },
```

Comment:
Nit: Extract this as DEFAULT_SORTING ? Isn't there alrady a default like this somewhere that we could reuse ?

Reply:
Good point! 👌